.tekton/rhtap-staging-e2e-ci.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: rhtap-staging-e2e-
+  annotations:
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "push" && target_branch == "main" &&
+      (
+        "**/staging/**".pathChanged() ||
+        "argo-cd-apps/app-of-app-sets/base/**".pathChanged() ||
+        "components/**/base/**".pathChanged()
```

Comment:
Isn't that going to trigger a test if `components/pipeline-service/production/base` is modified?

Wouldn't `"components/*/base/**"` be more appropriate?

Reply:
Hmm interesting but then  `"components/*/base/**"` also triggering pipeline on production base change. I have raised this concern to pac team on slack